### DISCUSSION TITLE
Problem with Greek

### DISCUSSION THREAD

**Discussion by mixtou** [posquit0/Awesome-CV#607] [General]
While greek language appears in general, however it is not displayed in cventries in %Location and %Date sections. Is there a problem with definition of \bodyfontlight ??

**Comment by posquit0** [comment]
I think you should change the fonts which support Greek characters.

**Comment by gpetrousov** [comment]
Hi @mixtou , I know this is quiet an old issue, however, here's a solution for the Greek language problem. You need to add the following lines. You can get an example from here https://github.com/gpetrousov/latex-thesis-template/blob/master/thesis.tex where I have created a template for writing thesis in Greek.

\usepackage{xgreek}
\setmainfont[Mapping=tex-text]{GFS Didot}

**Comment by chazarnik** [comment]
Actually that does not seem to work for me @gpetrousov . Did it work for you?

What worked after scanning the cls file was change the line below 
\newcommand*{\entrydatestyle}[1]{{\fontsize{8pt}{1em}\**bodyfont**\scshapehape\color{graytext} #1}}

from bodyfontlight and changing the next (shape part). 
Hope it helps!

**Comment by gpetrousov** [comment]
@Krooket - guess there are multiple ways to fix this problem, whatever works for everyone :relieved: In my case, I used the root file (tex) which bypasses the class file (cls) to define the fonts and typesetting which supports Greek.
